tests/tests_imuxsock_files.yml

```diff
@@ -60,7 +67,9 @@
       include_tasks: tasks/check_packages.yml
 
     - name: Check output to messages line
-      command: /bin/grep '\*.info;mail.none;authpriv.none;cron.none.*{{ __default_system_log }}' '{{ __test_files_conf }}'
+      command: "/bin/grep
```

Comment:
with `command` it is better to use `>-` which will generate a single line string with each newline replaced with a single space